plugin/js/parsers/BakaTsukiImageCollector.js

```diff
@@ -111,7 +112,18 @@ ImageElementConverter.prototype.replaceWithImagePageUrl = function (images) {
     let imageInfo = images.get(that.imagePageUrl);
     if (imageInfo != null && that.element.parentElement != null) {
         let newImage = imageInfo.createImageElement();
-        that.element.parentElement.replaceChild(newImage, that.element);
+        if(0 === 1){ // placeholder for some check if we should remove the duplicate images (currently resolves to false)
```

Comment:
Some variable that tells the epub gen to remove duplicate images. 
Resolves to false atm for obvious reasons.